### ISSUE TITLE
[Classic] Client Crash If In Party and uUsing Transports

### ISSUE THREAD

**Issue by Metalica** [cmangos/issues#296]
Priority:
- Critical
  
  What is the problem:
- When your in party and go from ratchet to booty bay. You or your friend client will crash.
- Zepelin is same story.
  
  How to re-make this bug report:
- Invite some one to party.
- Go alone or togheter from Ratchet to bootybay / You or him will have client crash).

**Comment by jimmybrancaccio** [comment]
I don't believe this is still an issue but I can test on WotLK. @Metalica Can you confirm which version you were seeing this issue on?

**Comment by Phatcat** [comment]
Using the same conditions for reproducing won't make it happen on tbc or wotlk, it is happening only on classic.

**Comment by Phatcat** [comment]
Not sure this is fixable it may be an issue with the client itself, this just happened to me on 1.13 retail.

**Comment by al3xc1985** [comment]
it's happening still on retail aswell. Texted a few days ago with the boat from darkshore to teldrassil.

We still need a way to fix this, if that is possible
